pkg/trace/local_client_test.go

```diff
@@ -1,82 +0,0 @@
-package trace
```

Comment:
this file was just renamed, not sure why github shows it as deleted sometimes and renamed in others